xt/lib/PageObject/Setup/CreateUser.pm

```diff
@@ -20,7 +20,7 @@ __PACKAGE__->self_register(
 
 my @fields = ("Username", "Password", "Create new user", "Import existing user",
               "Salutation", "First Name", "Last name", "Employee Number",
-              "Date of Birth", "Tax ID/SSN", "Country", "Assign Permissions");
+              "Country", "Date of Birth", "Tax ID/SSN", "Assign Permissions");
```

Comment:
Ideally, I'd like to see a test which implements filling out the country first and then the DOB *and* a test which fills out the DOB first and then the country. The inverted order shouldn't break anything.